internal/loggers/fields.go

```diff
@@ -33,10 +33,33 @@ func F(key string, value interface{}) Field {
 
 // CoordinateF builds a Field containing information taken from the provided coordinate
 func CoordinateF(coordinate coordinate.Coordinate) Field {
-	return Field{"coordinate", coordinate}
+	return Field{"coordinate",
+		struct {
+			Reference string
+			Project   string
+			Type      string
+			ConfigID  string
+		}{
+			coordinate.String(),
+			coordinate.Project,
+			coordinate.Type,
+			coordinate.ConfigId,
+		}}
 }
 
 // EnvironmentF builds a Field containing environment information for structured logging
-func EnvironmentF(environment string) Field {
-	return Field{"environment", environment}
+func EnvironmentF(environment, group string) Field {
```

Comment:
If you feel strongly about separating the logger from the monaco specifics like coordinate and the fact that "environment" information exists, it would have to move to a separate utility package that does just that. 

As is, log depends on loggers - and offers the FromCtx convenience - and fields are needed in the loggers interface. 
Moving either between packages to "encapsulate" the knowledge about monaco terms or objects in one of them the log fields will create a cyclic import. 

I am not convinced this separation of concerns is worth the added third `logutils` package tbh.

Reply:
Moved it to a log.field sub-package and also incorporated @jskelin's naming improvements with that :)